app/workers/hit_fox_feed_worker.rb

```diff
@@ -0,0 +1,12 @@
+require 'hit_fox_feed'
+
+class HitFoxFeedWorker
+  include Sidekiq::Worker
+  include Sidetiq::Schedulable
+
+  recurrence { minutely.second_of_minute(0, 5, 10, 15, 20, 25) }
```

Comment:
So I asked about this happening every 5 seconds on previous PRs. Did you test that it doesn't just happen for the first 25 seconds?